style.css

```diff
@@ -109,3 +104,268 @@ ul {
+  line-height: 24px;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  border: 1px solid #6070ff;
+  cursor: pointer;
+  color: #6070ff;
+  background-color: #fff;
+  height: 48px;
+  gap: 10px;
+  padding: 12px;
+  width: 124px;
+  margin-left: 40px;
+  border-radius: 9px;
+  letter-spacing: 0.03em;
+}
+
+.second_work_container {
+  display: grid;
+  height: 592px;
+  justify-content: center;
+  margin-top: 114px;
+  margin-left: 16px;
+  margin-right: 16px;
+  border-radius: 16px;
+}
+
+.nxt_wk {
+  display: flex;
+  width: 295px;
+  height: 220px;
+  margin-left: 40px;
+  margin-right: 40px;
+  border-radius: 8px;
+  padding-bottom: 12px;
+}
+
+.worktwo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 295px;
+  height: 80px;
+  margin-left: 40px;
+  margin-right: 40px;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.third_work_container {
+  display: grid;
+  height: 592px;
+  grid-column: auto;
+  justify-content: center;
+  margin-top: 114px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 16px;
+}
+
+.third_wk {
+  display: flex;
+  width: 295px;
+  height: 220px;
+  margin-left: 40px;
+  margin-right: 40px;
+  border-radius: 8px;
+}
+
+.workthree {
+  display: flex;
+  align-items: center;
+  width: 295px;
+  height: 80px;
+  margin-left: 40px;
+  margin-right: 40px;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.forth_work_container {
+  display: grid;
+  height: 592px;
+  grid-column: auto;
+  justify-content: center;
+  margin-top: 114px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 16px;
+}
+
+.forth_wk {
+  display: flex;
+  width: 295px;
+  height: 220px;
+  margin-left: 40px;
+  margin-right: 40px;
+  border-radius: 8px;
+  padding-bottom: 12px;
+}
+
+.workfour {
+  display: flex;
+  align-items: center;
+  width: 295px;
+  height: 80px;
+  margin-left: 40px;
+  margin-right: 40px;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.about_section_container {
+  display: flex;
+  flex-direction: column;
+  height: 1000px;
+  border-radius: 16px;
+  margin-left: 14px;
+  margin-right: 14px;
+}
+
+.about_my {
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  line-height: 52px;
+  font-size: 40px;
+  width: 200px;
+  height: 101px;
+  font-family: 'Poppins', sans-serif;
+  margin-top: 114px;
+  font-weight: 700;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.sec_abt {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 24px;
+  font-size: 16px;
+  margin-top: -5px;
+  height: 140px;
+  gap: 24px;
+  padding: 114px, 24px, 114px, 24px;
+  line-height: 24px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  width: 300px;
+}
```

Comment:
Please stick to Figma, like said the previous reviewer

<img width="362" alt="image" src="https://user-images.githubusercontent.com/7243132/170830511-021ae72a-8a34-48df-b4ba-65f24ca55799.png">

Figma:
<img width="362" alt="image" src="https://user-images.githubusercontent.com/7243132/170830522-45ff75d2-bf27-4b58-95a2-b9cd96169776.png">